k8s/k8s.go

```diff
@@ -444,6 +446,8 @@ type MigrationOps interface {
 	UpdateMigration(*v1alpha1.Migration) (*v1alpha1.Migration, error)
 	// DeleteMigration deletes the Migration
 	DeleteMigration(string) error
+	// ValidateMigration validate the Migration status
+	ValidateMigration(name string, namespace string, timeout, retryInterval time.Duration)
```

Comment:
Should return error. Travis is failing because of this.

Reply:
looks like my entire commit is missing, sorry about this will update